examples/niscope/plot-read-waveform.py

```diff
@@ -38,11 +38,21 @@
 
 channels = "0"
 
-def CheckStatus(scope_service, vi, result):
-    if (result.status != 0):
-        error_result = scope_service.GetErrorMessage(niscope_types.GetErrorMessageRequest(vi = vi, error_code = result.status))
-        print(error_result.error_message)
-        exit()
+# Checks for errors. If any, throws an exception to stop the execution.
+any_error = False
+def CheckForError (vi, status) :
+    global any_error
+    if(status != 0 and not any_error):
+        any_error = True
+        ThrowOnError (vi, status)
+
+def ThrowOnError (vi, error_code):
+    error_message_request = niscope_types.ErrorMessageRequest(
```

Comment:
Same as previous file, is change from `GetErrorMessage` to `ErrorMessage` API intentional?

Reply:
Done changing.